modules/test/playwright/tests/commerce/commerceProducts.spec.ts

```diff
@@ -0,0 +1,84 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2000 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+//@ts-ignore
+import {expect, mergeTests} from '@playwright/test';
+
+import {apiHelpersTest} from '../../fixtures/apiHelpersTest';
+import {commercePagesTest} from '../../fixtures/commercePagesTest';
+import {loginTest} from '../../fixtures/loginTest';
+
+export const test = mergeTests(apiHelpersTest, commercePagesTest, loginTest);
+
+test('LPD-5780 modal title and product name appear properly in product menu', async ({
+    apiHelpers,
+    commerceProductsPage
+}) => {
+
+    const catalog = await apiHelpers.headlessCommerceAdminCatalog.postCatalog(
+		'Product Catalog'
+	);
+
+    const product1 = await apiHelpers.headlessCommerceAdminCatalog.postProduct(
+		catalog.id,
+		'\"Product1\"'
```

Comment:
I think all of these post calls randomize the names by default. Do not provide hard-coded names if not required.

Reply:
Would it be preferred if I had it generate a randomized name and then added the unicode character? I just figured this was cleaner to hard-code so it's a bit more readable.